sql/_05_plcsql/_01_testspec/_02_declaration/_01_parameter/_05_in_out_mode/answers/09_default_value_param.answer

```diff
@@ -157,7 +157,7 @@ null
 
 
 a
-1
+'1'
```

Comment:
It has been reported as a bug. It would be better to wait for the processing result.
http://jira.cubrid.org/browse/CBRD-25750

Reply:
revert back.